### PR TITLE
Add packages that embeds resources in binary

### PR DESCRIPTION
**Pull request by faxal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ Join us on IRC at **#awesome-go** on freenode [web access](http://webchat.freeno
     - [Web Frameworks](#web-frameworks)
         - [Middlewares](#middlewares)
     - [Windows](#windows)
+    - [Resource Embedding](#resource-embedding)
```
Comment (shurcooL) [inline README.md]: This should be alphabetical.

File: README.md [hunk 2]
```diff
@@ -732,6 +733,12 @@ Software written in Go.
 * [webhook](https://github.com/adnanh/webhook) - Tool which allows user to create HTTP endpoints (hooks) that execute commands on the server
 * [Wide](https://wide.b3log.org) - A Web-based IDE for Teams using Golang.
 
+## Resource Embedding
+
+* [go.rice](https://github.com/GeertJohan/go.rice) - go.rice is a Go package that makes working with resources such as html,js,css,images and templates very easy. 
+* [bindata](https://github.com/jteeuwen/go-bindata) - Package that converts any file into managable Go source code. 
+* [go-resources](https://github.com/omeid/go-resources) - Unfancy resources embedding with Go.
+
```
Comment (shurcooL) [inline README.md]: This section is still not alphabetical. It should come after Package Management section and before Science and Data Analysis section.

Comment (mholt) [inline README.md]: And bindata should go before the other two, and go.rice should be last (e comes after i) - at least, I think so. :P

Comment (shurcooL) [inline README.md]: Is it `bindata` or `go-bindata`? /cc @jteeuwen

**Comment by mholt** [comment]
Perfect, thanks!

**Pull request merged**